APIBankingASP.NET/Models/FundsTransferByCustomerService2.cs

```diff
@@ -10,26 +10,21 @@ namespace FundsTransferByCustomerService2
     {
         public class BeneficiaryAddress
         {
-            public String address1 { get; set; }
-            public String address2 { get; set; }
-            public String address3 { get; set; }
-            public String postalCode { get; set; }
-            public String city { get; set; }
-            public String stateOrProvince { get; set; }
-            public String country { get; set; }
+            public String beneficiaryAddress1 { get; set; }
```

Comment:
why did you have to rename this ? the word 'beneficiary' is redundant/repeating..